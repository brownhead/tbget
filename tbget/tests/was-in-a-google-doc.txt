{'double encoding is great': 'super', 'traceback': {'traceback': 'Traceback (most recent call last):\n  File “tools/devshell.py”, line 145, in <module>\n    from devshell_eval import *\n  File
 “/Users/johnsullivan/khan/webapp/tools/devshell_eval.py”, line 4, in <module>\n    from assessment_items.models import *\n  File “/Users/johnsullivan/khan/webapp/assessment_items/models.py”, line 16, in <module>\n    import api.jsonify\n  File "/Users/johnsullivan/khan/webapp/api/jsonify.py", line 21, in <module>\n    import frozen_model\n  File "/Users/johnsullivan/khan/webapp/frozen_model.py", line 16, in <module>\n    import content.models\n  File “/Users/johnsullivan/khan/webapp/content/models.py”, line 77, in <module>\n    from intl import i18n\n  File "/Users/johnsullivan/khan/webapp/intl/i18n.py", line 43, in <module>\n    import intl.models\n  File "/Users/johnsullivan/khan/webapp/intl/models.py", line 16, in <module>\n    import intl.crowdin\n  File "/Users/johnsullivan/khan/webapp/intl/crowdin.py", line 15, in <module>\n    import intl.translate\n  File ‘/Users/johnsullivan/khan/webapp/intl/translate.py’, line 57, in <module>\n    import event_log\n  File "/Users/johnsullivan/khan/webapp/event_log.py", line 107, in <module>\n    from logging_tools import ua_logic\n  File "/Users/johnsullivan/khan/webapp/logging_tools/ua_logic.py", line 6, in <module>\n    from third_party import user_agents\n  File "/Users/johnsullivan/khan/webapp/third_party/__init__.py", line 137, in load_module\n    __import__(short_module_name)\n  File "/Users/johnsullivan/khan/webapp/third_party/user_agents/__init__.py", line 3, in <module>\n    from .parsers import parse\n  File "/Users/johnsullivan/khan/webapp/third_party/user_agents/parsers.py", line 4, in <module>\n    from ua_parser import user_agent_parser\n  File "/Users/johnsullivan/khan/webapp/third_party/ua_parser/user_agent_parser.py", line 484, in <module>\n    _v2_replacement))\n  File "/Users/johnsullivan/khan/webapp/third_party/ua_parser/user_agent_parser.py", line 46, in __init__\n    self.user_agent_re = re.compile(self.pattern)\n  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/re.py", line 190, in compile\n    return _compile(pattern, flags)\n  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/re.py", line 242, in _compile\n    p = sre_compile.compile(pattern, flags)\n  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_compile.py", line 498, in compile\n    p = sre_parse.parse(p, flags)\n  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 687, in parse\n    p = _parse_sub(source, pattern, 0)\n  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 308, in _parse_sub\n    itemsappend(_parse(source, state))\n  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 647, in _parse\n    p = _parse_sub(source, state)\n  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 308, in _parse_sub\n    itemsappend(_parse(source, state))\n  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 512, in _parse\n    if subpattern:\n  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 126, in __len__\n    def __len__(self):\nKeyboardInterrupt ', 'some_stuff': 'great', 'just splendid': 2}}


