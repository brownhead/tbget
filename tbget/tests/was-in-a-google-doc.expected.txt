Traceback (most recent call last):
  File "tools/devshell.py", line 145, in <module>
    from devshell_eval import *
  File "/Users/johnsullivan/khan/webapp/tools/devshell_eval.py", line 4, in <module>
    from assessment_items.models import *
  File "/Users/johnsullivan/khan/webapp/assessment_items/models.py", line 16, in <module>
    import api.jsonify
  File "/Users/johnsullivan/khan/webapp/api/jsonify.py", line 21, in <module>
    import frozen_model
  File "/Users/johnsullivan/khan/webapp/frozen_model.py", line 16, in <module>
    import content.models
  File "/Users/johnsullivan/khan/webapp/content/models.py", line 77, in <module>
    from intl import i18n
  File "/Users/johnsullivan/khan/webapp/intl/i18n.py", line 43, in <module>
    import intl.models
  File "/Users/johnsullivan/khan/webapp/intl/models.py", line 16, in <module>
    import intl.crowdin
  File "/Users/johnsullivan/khan/webapp/intl/crowdin.py", line 15, in <module>
    import intl.translate
  File "/Users/johnsullivan/khan/webapp/intl/translate.pyâ€™", line 57, in <module>
    import event_log
  File "/Users/johnsullivan/khan/webapp/event_log.py", line 107, in <module>
    from logging_tools import ua_logic
  File "/Users/johnsullivan/khan/webapp/logging_tools/ua_logic.py", line 6, in <module>
    from third_party import user_agents
  File "/Users/johnsullivan/khan/webapp/third_party/__init__.py", line 137, in load_module
    __import__(short_module_name)
  File "/Users/johnsullivan/khan/webapp/third_party/user_agents/__init__.py", line 3, in <module>
    from .parsers import parse
  File "/Users/johnsullivan/khan/webapp/third_party/user_agents/parsers.py", line 4, in <module>
    from ua_parser import user_agent_parser
  File "/Users/johnsullivan/khan/webapp/third_party/ua_parser/user_agent_parser.py", line 484, in <module>
    _v2_replacement))
  File "/Users/johnsullivan/khan/webapp/third_party/ua_parser/user_agent_parser.py", line 46, in __init__
    self.user_agent_re = re.compile(self.pattern)
  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/re.py", line 190, in compile
    return _compile(pattern, flags)
  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/re.py", line 242, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_compile.py", line 498, in compile
    p = sre_parse.parse(p, flags)
  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 687, in parse
    p = _parse_sub(source, pattern, 0)
  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 308, in _parse_sub
    itemsappend(_parse(source, state))
  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 647, in _parse
    p = _parse_sub(source, state)
  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 308, in _parse_sub
    itemsappend(_parse(source, state))
  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 512, in _parse
    if subpattern:
  File "/Users/johnsullivan/.virtualenv/khan27/lib/python2.7/sre_parse.py", line 126, in __len__
    def __len__(self):
KeyboardInterrupt ', 'some_stuff': 'great', 'just splendid': 2}}
